.cirrus.yml

```diff
@@ -250,6 +250,10 @@ task:
         # that depend on it.
         - ./script/tool_runner.sh make-deps-path-based --target-dependencies-with-non-breaking-updates
         - $PLUGIN_TOOL_COMMAND test --run-on-dirty-packages --exclude=script/configs/dart_unit_tests_exceptions.yaml
+    - name: go_router_dart_fix_test
+      test_script:
+        - cd packages/go_router/test_fix
+        - dart fix --compare-to-golden
```

Comment:
If you want custom, package-specific tests, they should be in your package, not in the CI config. The mechanism that's in place for that is the `custom-test` repo tooling command, which looks for a `tool/run_tests.dart` in your package, and runs it if it's there. `packages/cross_file/tool/run_tests.dart` is a simple example you can follow. (There's a legacy mode that uses a run_script.sh, but please do **not** add a new shell-based script. Those are only there because they predate standardizing on repo tooling.)

By default it will run on every host platform, so if you only run it on Linux you should check for `Platform.isLinux` in your script, and no-op if it's not Linux.